src/solver.rs

```diff
@@ -59,11 +59,11 @@ impl SolverConfig {
             },
             Ok(SMTSolver::CVC4) => SolverConfig {
                 cmd: cmd.into(),
-                args: vec!["--lang".into(), "smt2".into()],
+                args: vec!["--lang".into(), "smt2".into(), "--produce-models".into()],
```

Comment:
Is this now required in addition to setting the option inside the smtlib input?

Reply:
no, with this we don't have to put it in smtlib anymore